dotcom-rendering/src/components/MostViewedRight.tsx

```diff
@@ -29,47 +30,115 @@ interface Props {
 	stickToTop?: boolean;
 }
 
+interface MostViewedRightPayloadType {
+	tabs: FETrailTabType[];
+}
+
 export const MostViewedRight = ({
 	limitItems = 5,
 	stickToTop = false,
 }: Props) => {
+	type Variant =
+		| 'deeply-read-only'
+		| 'deeply-read-and-most-viewed'
+		| 'most-viewed-only'
+		| 'none';
+	const useDeeplyReadTestVariant = (): Variant => {
+		const ABTestAPI = useAB()?.api;
+		if (
+			ABTestAPI?.isUserInVariant(
+				'DeeplyReadRightColumn',
+				'deeply-read-only',
+			)
+		) {
+			return 'deeply-read-only';
+		} else if (
+			ABTestAPI?.isUserInVariant(
+				'DeeplyReadRightColumn',
+				'deeply-read-and-most-viewed',
+			)
+		) {
+			return 'deeply-read-and-most-viewed';
+		} else if (
+			ABTestAPI?.isUserInVariant(
+				'DeeplyReadRightColumn',
+				'most-viewed-only',
+			)
+		) {
+			return 'most-viewed-only';
+		} else {
+			return 'none';
+		}
+	};
+	const testVariant = useDeeplyReadTestVariant();
 	const endpointUrl =
-		'https://api.nextgen.guardianapps.co.uk/most-read-geo.json?dcr=true';
-	const { data, error } = useApi<FETrailTabType>(endpointUrl);
+		'https://api.nextgen.guardianapps.co.uk/most-read-with-deeply-read.json';
+	const { data, error } = useApi<MostViewedRightPayloadType>(endpointUrl);
 
 	if (error) {
-		window.guardian.modules.sentry.reportError(error, 'most-viewed-right');
+		window.guardian.modules.sentry.reportError(
+			error,
+			'most-viewed-deeply-read-right',
+		);
 		return null;
 	}
 
 	if (data) {
-		const trails: TrailType[] = data.trails
-			.slice(0, limitItems)
-			.map(decideTrail);
+		const mostReadTrails = data.tabs[0]?.trails
+			?.slice(0, limitItems)
+			?.map(decideTrail);
+		const deeplyReadTrails = data.tabs[1]?.trails
+			?.slice(0, limitItems)
+			?.map(decideTrail);
```

Comment:
If we're not in a test where we render this data, this work is entirely unnecessary.